jobbergateapi2/tests/apps/users/test_routers.py

```diff
@@ -99,15 +100,56 @@ async def test_create_user_duplication(client, user_data):
 
 @pytest.mark.asyncio
 @database.transaction(force_rollback=True)
-async def test_create_user_admin(client, user_data):
+async def test_create_superuser(client, user_data):
     """
-    Test creating the user with the admin permission
+    Test create a superuser.
     """
-    user_data["is_admin"] = True
+    user_data["is_superuser"] = True
     response = client.post("/users/", json=user_data)
     assert response.status_code == status.HTTP_200_OK
     count = await database.fetch_all("SELECT COUNT(*) FROM users")
     assert count[0][0] == 1
 
     user = User.parse_obj(await database.fetch_one(users_table.select()))
-    assert user.is_admin is True
+    assert user.is_superuser is True
+
+
+@pytest.mark.asyncio
+@database.transaction(force_rollback=True)
+async def test_users_get(client, user_data):
+    """
+    Test getting a user by id.
+    """
+    user = [UserCreate(id=1, **user_data)]
+    await insert_objects(user, users_table)
+
+    response = client.get("/users/1")
+    assert response.status_code == status.HTTP_200_OK
+
+    data = response.json()
+    assert data["email"] == user_data["email"]
+
+
+@pytest.mark.asyncio
+async def test_users_get_empty(client):
+    """
+    Try to get the user of id=1 which doesn't exists and will return 404.
+    """
+    response = client.get("/users/1")
```

Comment:
did we not just create a user with id=1 in the previous test?

Reply:
It was created, but every test is in its own empty environment, after a test is ran, the database is cleared, that way a test don't interfere with another